doc/source/release-notes/Percona-Server-5.7.23-23.rst

```diff
@@ -0,0 +1,75 @@
+.. rn:: 5.7.22-22
+
+========================
+Percona Server |release|
+========================
+
+Percona is glad to announce the release of Percona Server |release| on
+August 30, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.23-23/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.23
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-23.html>`_
+and includes all the bug fixes in it. |Percona Server| |release| is
+now the current GA (Generally Available) release in the 5.7 series.
+
+All software developed by Percona is open-source and free.
+
+New Features
+============
+
+* The :variable:`max_binlog_files` variable is deprecated and replaced
+  with with the :variable:`binlog_space_limit` variable. The behavior of
+  :variable:`binlog_space_limit` is consistent with the variable
+  :variable:`relay-log-space-limit` used for relay logs and having the same
+  semantics.
+* In the InnoDB system tablespace all data are encrypted including the parallel
+  double write buffer. A new variable :variable:`innodb_sys_tablespace_encrypt`
+  is introduced to encrypt the system tablespace . The encryption of the
+  parallel double write buffer file is controlled by the variable
+  :variable:`innodb_parallel_dblwr_encrypt`. Both variables are ``OFF`` by
+  default.
+* Changing ``rocksdb_update_cf_options`` shows warnings and errors in the
+  client.
+	
+Bugs Fixed
+==========
+
+* The statement ``SELECT...ORDER BY`` produced inconsistent results with the
+  ``euckr`` charset or ``euckr_bin`` collation. Bug fixed :psbug:`4513`
+  (upstream :mysqlbug:`91091`).
+* InnoDB statistics could incorrectly report zeros in the slow query log. Bug
```

Comment:
InnoDB extensions for the slow query log, consider linking to the feature doc page